cmd/operator-ui/actions/blackducks.go

```diff
@@ -218,7 +218,7 @@ func (v BlackducksResource) common(c buffalo.Context, bd *blackduckapi.Blackduck
 
 	// environment variables
 	if bd.View.Environs == nil {
-		env := containers.GetHubKnobs()
+		env := containers.GetHubKnobs(bd.Spec.Version)
```

Comment:
Why is this now needed?

Reply:
we changed GetHubKnobs method to get the version as input to support SAML verification